go.work

```diff
@@ -4,4 +4,5 @@ use (
 	.
 	schema
 	test
+	test/vendorproj
```

Comment:
For the record: I added this submodule to test if the scripts allow having a mixed project, with some modules having `vendor/` and some don't.